playwright-gh-pages/deploy-report-pages/action.yml

```
@@ -0,0 +1,88 @@
+name: 'Publish to GH pages'
+description: 'Publishes test artifacts to GitHub Pages and comments on the PR with the results and links.'
+author: 'frontend@grafana'
+
+inputs:
+  github-token:
+    description: 'Token for the repository. Can be passed in using `{{ secrets.GITHUB_TOKEN }}`.'
+    required: true
+  retention-days:
+    description: 'Number of days to retain the reports. Default is 30.'
+    required: true
+    default: 30
+  pr-comment-summary:
+    description: 'Whether to comment the PR with the test results. Default is true.'
+    required: true
+    default: true
+  artifact-pattern:
+    description: 'Pattern to match the artifacts. Default is "playwright-report-*"'
+    required: true
+    default: 'playwright-report-*'
+  
+runs:
+  using: "composite"
+  steps:
+    - name: Download report artifacts
+      uses: actions/download-artifact@v4
+      with:
+        path: all-reports
+        pattern: ${{ inputs.artifact-pattern }}
+        merge-multiple: true
+
+    - name: Delete any existing comment
+      uses: thollander/actions-comment-pull-request@v3
+      with:
+        comment-tag: test-results
```

this tag might be too genereric
```suggestion
        comment-tag: gf-playwright-test-results
```